lib/Conversion/ImportVerilog/Expressions.cpp

```diff
@@ -77,7 +84,9 @@ struct ExprVisitor {
   // Handle blocking and non-blocking assignments.
   Value visit(const slang::ast::AssignmentExpression &expr) {
     auto lhs = context.convertExpression(expr.left());
+    context.lvalueStack.push_back(lhs);
     auto rhs = context.convertExpression(expr.right());
+    context.lvalueStack.pop_back();
```

Comment:
There is no test that really exercises this stack, i.e., a case where more than one value is on the stack.

Reply:
Great point. I'll add one.